src/core/importType.js

```diff
@@ -8,8 +8,8 @@ function constant(value) {
   return () => value
 }
 
-function baseModule(name) {
-  if (isScoped(name)) {
+function baseModule(name, path) {
+  if (isScoped(name, path)) {
```

Comment:
i don't think it makes sense to make `isScoped` check `isExternalPath`; i think all the places currently calling `isScoped` should be explicitly handling `isExternalPath` first.

Reply:
I can do that. It does seem that "scoped" packages have a connotation meaning "external". So I thought it would be fine putting it in `isScoped` because scoped modules have to be external. Will update shortly

Reply:
Actually... I don't even know how to solve this? Move the `isScoped` check down underneath `isInternalModule`in the lodash.cond call? The problem I am seeing is that isScoped means external, but isExternalModule doesn't check it. It is checked by the lodash.cond call

Reply:
Seems like moving the internal check above scoped and external solved that problem and let me revert most of the other changes

Reply:
Scoped packages are the same as anything in node_modules; they're things installed from npm. I don't think "scoped" means "external".